virttest/utils_misc.py

```diff
@@ -2056,6 +2056,34 @@ def compare_qemu_version(major, minor, update, is_rhev=True, params={}):
     return True
 
 
+def is_qemu_feature_supported(params, ignore_error=False):
+    """
+    Check whether the function is supported in this qemu-kvm version by comparing
+    the installed qemu-kvm version
+
+    :param params: Dictionary with the test parameters
+    :param ignore_error: Whether to raise an exception
+    :raise: When ignore_error is set False, raise TestCancel if the feature is
+        not supported
+    :return: True if the feature is supported;
+        False if the feature is not supported when ignore_error is set to True
+    """
+    supported_qemu_ver = eval(
+        params.get("func_supported_since_qemu_kvm_ver", '()'))
+    unsupported_err_msg = params.get("qemu_unsupported_err_msg",
+                                     "This qemu version doesn't support "
+                                     "this function.")
+
+    if supported_qemu_ver:
+        if not compare_qemu_version(*supported_qemu_ver, False):
+            if ignore_error:
```

Comment:
From function name implication meaning, not suggest add additional parameter : ignore_error to enable exceptions.TestCancel. Better just return pure bool value:True or False. You may use another high level function to enable exceptions.TestCancel

Reply:
I think this function can be a high level qemu version comparing function. We don't need to add another to raise cancel exception I think. And it also can return True/False, so we can continue to check other things if needed.  What do you think?